Add guideline to disallow fix bundling and excessively large PR (#16686)
ParadiseSS13/Paradise: .github/CONTRIBUTING.md

 of a pull request needs to be removed/changed, you may simply modify that single commit.
 Due to limitations of the engine, this may not always be possible; but do try your best.

  * Keep your pull requests small and reviewable whenever possible. Do not bundle unrelated

>  fix even if it generate more pull requests otherwise. In case of mapping PR to add in a feature -

Reviewer: ```suggestion
 fixes even if that generates more pull requests. In case of a mapping PR to add in a feature -
```

>  fix even if it generate more pull requests otherwise. In case of mapping PR to add in a feature -

Reviewer 2: ```suggestion
 fixes even if not bundling it generates more pull requests. In case of mapping PR to add in a feature -
```

>  fixes even if not bundling it generates more pull requests. In case of mapping PR to add in a feature -

Reviewer: ```suggestion
 fixes even if not bundling them generates more pull requests. In case of mapping PRs that add features -
```

>  consult a maintainer on whether it would be appropriate to split up the PRs to add the

Reviewer: ```suggestion
 consult a maintainer on whether it would be appropriate to split up the PR to add the
```
